sdk/bulk.go

```diff
@@ -0,0 +1,150 @@
+package sdk
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/opentdf/platform/protocol/go/kas"
+)
+
+// BulkTDF: Reader is TDF Content. Writer writes encrypted data. Error is the error that occurs if decrypting fails.
+type BulkTDF struct {
+	Reader io.ReadSeeker
+	Writer io.Writer
+	Error  error
+}
+
+type BulkDecryptRequest struct {
+	TDFs    []*BulkTDF
+	TDFType TdfType
+}
+
+// BulkErrors List of Errors that Failed during Bulk Decryption
+type BulkErrors []error
+
+func (b BulkErrors) Error() string {
```

Comment:
Optional? I don't think this really is an Unwrap? But it might make sense to implement [the unwrap interface](https://pkg.go.dev/errors#pkg-overview) anyway

```suggestion
func (b BulkErrors) Unwrap() []error {
	return b
}

func (b BulkErrors) Error() string {
```